pages/express-relay/integrate-as-searcher.mdx

```diff
@@ -0,0 +1,258 @@
+
+Searchers **bid** on opportunities exposed by the Express Relay server.
+The server exposes different endpoints for interaction, which can be used directly via HTTP, WebSocket, or one of the SDKs for convenience.
+
+Searchers can integrate with the Express Relay server in three steps:
+
+1. Subscribe to new opportunities
+2. Evaluate the opportunity and construct the bid
+3. Submit the bid to the auction server
+
+<Steps>
+
+### Subscribing to new opportunities
+
+Express Relay provides searchers with `Typescript` and `Python` SDKs to interact with the server. 
+Searchers can also fetch available opportunities via HTTP or subscribe to them via WebSocket.
+
+<Tabs items={['Typescript', 'Python', 'HTTP', 'Websocket']}>
+
+<Tabs.Tab>
+Our JavaScript SDK provides a convenient way to subscribe to opportunities:
+
+```typescript
+import { Client, Opportunity } from "@pythnetwork/express-relay-evm-js";
+
+const handleOpporunity = async (opportunity: Opportunity) => {
+  // TODO: Implement your opportunity handler here
+};
+
+const client = new Client(
+  { baseUrl: "https://pyth-express-relay-mainnet.asymmetric.re" },
+  undefined,
+  handleOpporunity
+);
+await client.subscribeChains(["op_sepolia"]);
+```
+
+</Tabs.Tab>
+<Tabs.Tab>
+    Our Python SDK provides a convenient way to subscribe to opportunities:
+
+```python copy
+from express_relay.client import (
+    ExpressRelayClient,
+)
+from express_relay.express_relay_types import Opportunity
+
+
+def opportunity_callback(opportunity: Opportunity):
+    # TODO: Implement your opportunity handler here
+    pass
+
+
+client = ExpressRelayClient(
+    'server_url',
+    None,
+    opportunity_callback,
+    None,
+)
+await client.subscribe_chains(['op_sepolia'])
+```
+
+</Tabs.Tab>
+<Tabs.Tab>
+Searchers can request opportunities through an HTTP GET call to the `/v1/opportunities` endpoint.
+
+```bash copy
+curl -X 'GET' \
+  'https://pyth-express-relay-mainnet.asymmetric.re/v1/opportunities?chain_id=op_sepolia&mode=live'
+```
+
+Opportunities are short-lived and will be executed in a matter of seconds. So, the above endpoint can return an empty response.
+
+You can fetch historical opportunities by setting the `mode` parameter to `historical`.
+
+</Tabs.Tab>
+<Tabs.Tab>
+Searchers can connect to the server via WebSocket to reduce latency and subscribe to various events. The WebSocket endpoint relies at `/v1/ws`(e.g `wss://pyth-express-relay-mainnet.asymmetric.re/v1/ws`).
+Here is a sample JSON payload to subscribe to opportunities:
+
+```bash copy
+{
+  "id": "1",
+  "method": "subscribe",
+  "params": {
+    "chain_ids": ["op_sepolia"]
+  }
+}
+```
+
+Consult [`Websocket API reference`](./websocket-api-reference.mdx) for a complete list of methods and parameters.
+
+</Tabs.Tab>
+</Tabs>
+
+The server responds with opportunities in the following format:
+
+```bash copy
```

Comment:
Update the comments in this JSON file

Reply:
I took the same comments from the protocol guide in the new update.